packages/core/src/transports/base.ts

```diff
@@ -0,0 +1,11 @@
+import * as init from '../initialize';
+import type { Agent } from '../types';
+import type { Transport, TransportItem } from './types';
+
+export abstract class BaseTransport implements Transport {
```

Comment:
Should `BaseTransport` also contain `name` and `version`?

Reply:
we don't have a use for  it (yet) :thinking: